tests/pom.xml

```diff
@@ -802,7 +802,7 @@
                       <time>${service.startup.timeout}</time>
                       <http>
                         <method>GET</method>
-                        <url>http://${auth.ip}:${auth.health.port}/readiness</url>
+                        <url>http://${auth.ip}:${auth.health.port}/q/health/started</url>
```

Comment:
The Auth Server does not register any readiness checks so that check would yield a 200 OK right from the beginning. We could replace it with `/q/health` which is an aggregation of all checks, i.e. started AND ready AND live. I have also used that endpoint in the other components.

Reply:
Ok. Either one of `q/health/started` and `/q/health` is ok for me here then.

Reply:
Then maybe `/q/health` would indeed be a better choice as this endpoint will indicate that the component is ready and live (implying that it has also been started successfully).

Reply:
I have pushed a corresponding change

Reply:
> Then maybe `/q/health` would indeed be a better choice

Right. It's also more future-proof - in case some readiness checks get added later.

Reply:
(see other discussion regarding this file below)